pkg/clusterinit/onboard/passthrough.go

```diff
@@ -49,44 +50,43 @@ func (s *passthroughGenerator) Patches() []cinitmanifest.Patch {
 
 func (s *passthroughGenerator) Generate(ctx context.Context, log *logrus.Entry) (map[string][]interface{}, error) {
 	clusterRoot := BuildFarmDirFor(s.clusterInstall.Onboard.ReleaseRepo, s.clusterInstall.ClusterName)
-
-	subFS, err := fs.Sub(s.manifests, passthroughRoot)
-	if err != nil {
-		return nil, fmt.Errorf("subfs: %w", err)
-	}
-
 	pathToManifests := make(map[string][]interface{})
 
-	if err := fs.WalkDir(subFS, ".", func(p string, d fs.DirEntry, _ error) error {
-		if p == "." || d.IsDir() {
-			return nil
-		}
-
-		bytes, err := s.readFile(subFS, p)
+	for _, v := range s.manifests {
+		subFS, err := fs.Sub(v, passthroughRoot)
```

Comment:
I'm not sure that `passthroughRoot` (which value is `manifests`) is right for each `FS`. We have one filesystem rooted at `manifests` but the new one is `multiarchbuildconfig/v1/ci.openshift.io_multiarchbuildconfigs.yaml`.